swarm.go

```diff
@@ -60,6 +61,8 @@ type Swarm struct {
 		m map[peer.ID][]*Conn
 	}
 
+	stripedPeerConnectivity [256]sync.Mutex
```

Comment:
pull out the number of stripes to a constant

Reply:
Done.